### ISSUE TITLE
Mekanism copper ingot gear bug

### ISSUE THREAD

**Issue by felix166** [AllTheMods/ATM-3#861]
## Expected Behavior
You can make Thermal Expansion gears with Mekanism copper.

## Current Behavior
You can't make Thermal Expansion Gears with Mekanism Copper.

## Possible Solution
Maybe repair the recipe, because it works on other Modpacks....

## Steps to Reproduce

1. take 4 mekanism copper
2. go to a crafting bench
3. place it like so:
![2019-04-23_12 47 03](https://user-images.githubusercontent.com/49905883/56576076-bb7dec80-65c7-11e9-88fc-91c04f807c3c.png)

## Detailed Description
I like the ore processing from Mekanism, and if want to process copper ore i get this copper.
With this copper i would like to make Thermal Expansion gears.

**Comment by martmists** [comment]
Does this recipe exist in JEI? If not, this is intended.

**Comment by felix166** [comment]
No, It doest exist. Why does this should be intended? You can make these gears with normal copper, but why not with these?
![2019-04-23_13 49 28](https://user-images.githubusercontent.com/49905883/56578734-a22c6e80-65ce-11e9-8723-75099cf00e26.png)

**Comment by FallenEagle1** [comment]
The intent was that unidict wouldve fixed these recipes, but it almost seems like it is still broken. It broke on an update and been seeing the effects since. Which modpack version are you on? That could make a difference.

**Comment by felix166** [comment]
I´m playing on All the Mods 3 5.12.1

**Comment by alcheya** [comment]
Is there a temp fix for this? I'm experiencing the same issue with the latest release.

**Comment by Darkere** [comment]
Use something to convert the Ingot from Mekanism to Thermal. The forge lexicon can do it and thermal also adds a block that does it.

**Comment by Yamza** [comment]
Forge Lexicon is in the Akashic Tome.

**Comment by alcheya** [comment]
Thanks folks

**Comment by Dviper18** [comment]
Encounted same bug with trying to make Invar blend using Mekanism Iron dust. (While in crafting grid)
Put both in an Induction Smelter and it makes Invar ingots.

**Comment by FallenEagle1** [comment]
Reopening to remind myself. It seems like something with our script is having issues, so need to dig into this deeper.
